nebuly-platform/templates/_batch_jobs.tpl

```diff
@@ -83,25 +83,25 @@
 - name: INTENT_LOCAL_MODEL_NAME
   value: "intent-classifier"
 - name: INTENT_LOCAL_MODEL_VERSION
-  value: "1"
+  value: "2"
 - name: INTENT_CLASSIFIER_KIND
   value: "local_model_majority_vote"
 
 
 # Sub Intent Settings
 - name: SUB_INTENT_LOCAL_MODEL_NAME
-  value: "topic-classifier"
+  value: "intent-classifier"
 - name: SUB_INTENT_LOCAL_MODEL_VERSION
-  value: "1"
+  value: "2"
 - name: SUB_INTENT_CLASSIFIER_KIND
   value: "local_model_majority_vote"
 
 
 # Sub Sub Intent Settings
 - name: SUB_SUB_INTENT_LOCAL_MODEL_NAME
-  value: "topic-classifier"
+  value: "intent-classifier"
 - name: SUB_SUB_INTENT_LOCAL_MODEL_VERSION
-  value: "1"
+  value: "2"
```

Comment:
Is this model identical to the intent classifier? If so, we should retrieve the version and name from the .Values, otherwise it becomes extremely error prone

Reply:
@Telemaco019  I am not an expert of this syntax i just copied what was done for other fields in the same yaml. 
Have a look.

Reply:
Just noticed something. 
I added the | quote everywhere since it was done this way in the other parts of the yaml. 
But in the values i already have the quotes: 

`
# -- Settings of the AI models used for inference.
aiModels:
  # -- The kind of registry used to store the AI models.
  # -- Available values are: "azure_ml", "aws_s3", "azure_storage", "gcp_bucket"
  registry: ""
  modelEmbeddingWarnings:
    name: "warning-embedding"
    version: 1
  modelEmbeddingIntents:
    name: "intent-embedding"
    version: 3
  modelEmbeddingTopic:
    name: "topic-embedding"
    version: 4
  modelInferenceInteractions:
    name: "interaction-analyzer-7b-v2"
    version: 16
  modelIssuesClassifier:
    name: "model-issues-classifier"
    version: "2"

  modelIntentClassifier:
    name: "intent-classifier"
    version: "1"
    version: "2"

  modelTopicClassifier:
    name: "topic-classifier"
    version: "1"
  modelActionClassifier:
    name: "action-classifier"
    version: "1"
`

should i remove the quotes from here and leave the version as int?  don't know if it is an issue anyway the field name as the quotes everywhere. 

Sorry for the dumb question but i don't know how this works.